src/search.cpp

```diff
@@ -446,9 +446,11 @@ void Search::Worker::iterative_deepening() {
             double reduction = (1.48 + mainThread->previousTimeReduction) / (2.17 * timeReduction);
             double bestMoveInstability = 1 + 1.88 * totBestMoveChanges / threads.size();
             int    el                  = std::clamp((bestValue + 750) / 150, 0, 9);
+            double recapture =
+              limits.capSq != SQ_NONE && limits.capSq == rootMoves[0].pv[0].to_sq() ? 0.955 : 1.005;
```

Comment:
dont think we need the `limits.capSq != SQ_NONE` check? It is already implied by `limits.capSq == rootMoves[0].pv[0].to_sq()` and here we should always have a valid pv move.

Reply:
Yes, good spot. Testing for the second element (ponder) makes sense, but I guess the first one has to be there!